integration_tests/helpers/ffi/wallet.js

```diff
@@ -321,6 +337,22 @@ class Wallet {
     return result;
   }
 
+  getBalance() {
+    return this.balance;
+  }
+
+  pollBalance() {
```

Comment:
I think this method would be more aptly named `getBalanceFromWallet()` to minimize confusion (see related comment)